.github/workflows/gridproxy-release.yml

```
@@ -42,6 +42,7 @@ jobs:
       - name: Build and push Docker image
         uses: docker/build-push-action@v4
         with:
+          context: "{{defaultContext}}:grid-proxy"
```

just the path to the dockerfile should work
```suggestion
          context: "grid-proxy"
```